include/openspace/scene/asset.h

```diff
@@ -207,6 +207,14 @@ class Asset {
      */
     bool hasInitializedParent() const;
 
+    /**
+     * Returns a list of the parents of this Asset that is currently in an initialized
+     * state, meaning any parent that is still interested in this Asset at all.
+     *
+     * \return list of parent filepaths that are interested in this asset, empty otherwise
```

Comment:
```suggestion
     * \return A list of parent filepaths that are interested in this asset
```

The "empty otherwise" part did not quite make sense grammatically. Either expand it so that it makes sense (e.g. "An empty list if there are no interested parents"). But I think that's alrady implied by just the first part :)